src/Main.java

```diff
@@ -1,5 +1,24 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+            Author stephen = new Author("Stephen", "King");
+            Author Joanne = new Author("Joanne", "Rowling");
```

Comment:
```suggestion
            Author joanne = new Author("Joanne", "Rowling");
```
названия переменных надо писать с маленькой буквы